pom.xml

```diff
@@ -447,6 +447,85 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>check</id>
+      <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <groupId>org.eclipse.smarthome.tools</groupId>
+              <artifactId>static-code-analysis</artifactId>
+              <version>${project.version}</version>
+              <executions>
+                <execution>
+                  <phase>verify</phase>
+                  <goals>
+                    <goal>checkstyle</goal>
+                    <goal>pmd</goal>
+                    <goal>findbugs</goal>
+                    <goal>report</goal>
+                  </goals>
+                </execution>
+              </executions>
+            </plugin>
+          </plugins>
+        </pluginManagement>
+      </build>
+    </profile>
+    <profile>
+      <id>check-bundles</id>
+      <activation>
+        <file>
+          <exists>META-INF/MANIFEST.MF</exists>
+        </file>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.eclipse.smarthome.tools</groupId>
+            <artifactId>static-code-analysis</artifactId>
+            <version>${project.version}</version>
+            <configuration>
+              <ruleset>bundle</ruleset>
+              <findbugsPlugins>
+                <findbugsPlugin>
```

Comment:
Shouldn't this plugin added internal in your static-code-analysis artifact.
I don't think it should be an optional plugin that could be added optional on `static-code-analysis` plugin usage.
It should be a mandatory findbugs plugin which is added by the plugin itself internal.